modules/virt-copying-the-ssh-command.adoc

```diff
@@ -4,12 +4,15 @@
 
 :_content-type: PROCEDURE
 [id="virt-copying-the-ssh-command_{context}"]
-= Copying the SSH command from the web console
+= Copying the SSH command using the web console
+
+Copy the command to connect to a virtual machine (VM) terminal via SSH. The web console allows you to quickly access the VM terminal by using `virtctl`.
 
-Copy the command to access a running virtual machine (VM) via SSH.
 
 .Procedure
 
 . In the {product-title} console, click *Virtualization* -> *VirtualMachines* from the side menu.
-. Use the search and filters to find your VM in the list.
-. Click the *Options* menu {kebab} for your virtual machine and select *Copy SSH command*. You can now paste this command onto the OpenShift CLI (oc).
+
+. Click the *Options* menu {kebab} for your virtual machine and select *Copy SSH command*.
+
+. Paste it in the OpenShift CLI (`oc`) to access the VM terminal.
```

Comment:
Is there a reason why this is phrased as "Paste it in the OpenShift CLI (`oc`)" instead of telling the user to paste it into a terminal (generic)?

Reply:
That was how it was phrased originally. If you think it might be more accurate to say "terminal" instead of the OpenShift CLI, I am fine with making that change.